### DISCUSSION TITLE
Where is the app?

### DISCUSSION THREAD

**Discussion by 0belous** [S-CODE-pl/ChainBreaker#2] [General]
I cannot for the life of me find where to actually use the thing, Im probably just being blind but i cant find any links anywhere and searching on the play store isn't any better.

**Comment by S-CODE-pl** [comment]
Hi, No, you're not blind :) App is in development, watch repo to get notify when it will be available for download!

**Comment by Damien0505** [comment]
Hello,
I was curious to know how things are progressing, and to let you know I'm happy to assist with beta testing!  :)

**Comment by Blingbug** [comment]
Please don't abandon this project, the world needs this!

**Reply by S-CODE-pl** [reply]
The project is not abandoned :), just slightly delayed. - requires more work than was estimated at the beginning

**Reply by Blingbug** [reply]
oh thats wonderful news! thank you

**Comment by Jaaa67** [comment]
Hi, i would really like to test the beta.
I hope the progress is going well, and there will be a app soon.

**Comment by S-CODE-pl** [comment]
we finally have a public version of the application, take a look https://play.google.com/store/apps/details?id=s_code.pl.chainbreaker

**Reply by Damien0505** [reply]
Here I was hoping this was going to be an amazing open source project, I check it out on Play Store, and it is AU$17.99?!  A bit steep for an app that is an unknown quantity.
